EIPS/eip-1313.md

```diff
@@ -0,0 +1,62 @@
+---
+eip: 1313
+title: Soulmates Token
+description: Bind souls(魂), tokens representing mates.
+author: Jun Kawasaki
+discussions-to: [https://ethereum-magicians.org/t/eip-5114-soulbound-token/9417](https://ethereum-magicians.org/t/eip-1313-soulmate-token/10132)
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-07-27
+requires: 721, 5114
+---
+
+## Abstract
+
+A soulmates token is a countable soulbound token. when it is minted, and cannot be transferred/moved after that. but its can be add new mate anytime.
+
+## Specification
+```solidity
+interface IERC1313 {
+	// fired anytime a new instance of this token is minted
+	// this event **MUST NOT** be fired twice for the same `tokenId`
+	event Mint(uint256 indexed tokenId, address indexed nftAddress, uint256 indexed nftTokenId);
+	
+  // returns the NFT token that owns this token.
+	// this function **MUST** throw if the token hasn't been minted yet
+	// this function **MUST** always return the same result every time it is called after it has been minted
+	// this function **MUST** return the same value as found in the original `Mint` event for the token
+	function ownerOf(uint256 index) external view returns (address nftAddress, uint256 nftTokenId);
+  
+  // returns a current number of mates.
+  // this function **MUST** return the latest number of tokens.
+  function current(Counter storage counter) internal view returns (uint256)
```

Comment:
```suggestion
  function current(uint256 counter) internal view returns (uint256)
```
Counter must be defined if it is to be used. `uint256` or other primitives are preferred.